database/database.go

```diff
@@ -0,0 +1,76 @@
+package database
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+
+	"final_project/tests"
+
+	_ "modernc.org/sqlite"
+)
+
+var ActualDbPath string
+
+// InitializeDB проверяет существование базы данных, создаёт её и таблицы при необходимости.
+func InitializeDB() error {
+	// Используем путь из переменной окружения или из тестового файла по умолчанию
+	dbPath := tests.DBFile
+	if path := os.Getenv("TODO_DBFILE"); path != "" {
+		dbPath = path
+	}
+
+	ActualDbPath = dbPath
+
+	// Проверяем существование файла базы данных
+	_, err := os.Stat(dbPath)
+	if os.IsNotExist(err) {
+		// Если базы данных нет, создаём её и таблицы
+		err = createAndInitializeDB(dbPath)
+		if err != nil {
+			return fmt.Errorf("Ошибка при создании базы данных: %v", err)
```

Comment:
При выводе в лог `%v`, а при оборачивании `%w` (wrap)
```suggestion
			return fmt.Errorf("Ошибка при создании базы данных: %w", err)
```

Reply:
А так за оборачивание ошибки плюс

Reply:
Эту ошибку много где нужно исправить